### PR TITLE
Package scripts for Arch Linux

### PR DESCRIPTION
**Pull request by ghost**
'ebusd' - PKGBUILD for released version of ebusd.
'ebusd-git' - PKGBUILD for GitHub master.

See https://wiki.archlinux.org/index.php/PKGBUILD for more details, if needed.

The user contributed PKGBUILDs for Arch Linux are usually maintained within AUR repository of Arch web site, however due to niche market of ebusd, I expect upstream to be a better place.

### CODE REVIEW

File: contrib/archlinux/ebusd/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+# Contributor: Milan Knizek <knizek volny cz>
+
+pkgname=ebusd
+#_gitname=ebusd
+pkgver=1.0.0
+_pkgver=1.0.0
+pkgrel=1
+pkgdesc="ebusd provides a communication interface to the energy bus (ebus)."
+arch=('i686' 'x86_64' 'armv6h')
+url=https://github.com/yuhu-/ebusd/
```
Comment (john30) [inline contrib/archlinux/ebusd/PKGBUILD]: this should read "https://github.com/john30/ebusd/"

Comment (ghost) [inline contrib/archlinux/ebusd/PKGBUILD]: Sorry, I will create a new request.

File: contrib/archlinux/ebusd-git/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+license=('GPL3')
+depends=()
+makedepends=('git')
+optdepends=()
+conflicts=('ebusd')
+provides=('ebusd')
+install=ebusd.install
+options=()
+backup=('etc/conf.d/ebusd')
+source=('git://github.com/john30/ebusd.git' 'ebusd.service' 'ebusd_conf.d')
```
Comment (john30) [inline contrib/archlinux/ebusd-git/PKGBUILD:21]: this is different from the ebusd/PKGBUILD line ('ebusd_conf.d' instead of 'ebusd'). Is that correct?

Comment (ghost) [inline contrib/archlinux/ebusd-git/PKGBUILD:21]: The correct one is 'ebusd_conf.d'. I have updated the contrib/archlinux/ebusd/PKGBUILD, tested that both release and git versions compile (makepkg -s) and will create a new pull request soon.

Thanks for noticing.

**Pull request merged**